opensuse-tomcat-image/pom.xml

```diff
@@ -0,0 +1,125 @@
+    <packaging>pom</packaging>
+
+    <name>openSUSE Tomcat image</name>
+
+    <properties>
+        <maven.install.skip>true</maven.install.skip>
+        <skipNexusStagingDeployMojo>true</skipNexusStagingDeployMojo>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-access</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.github.cafapi.logging</groupId>
+            <artifactId>caf-logging-logback-access</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.github.cafapi.opensuse.tomcat</groupId>
+            <artifactId>caf-logging-tomcat-juli</artifactId>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>io.fabric8</groupId>
+                <artifactId>docker-maven-plugin</artifactId>
+                <version>${fabric8.docker.maven.version}</version>
+                <executions>
+                    <execution>
+                        <id>docker-build</id>
+                        <phase>compile</phase>
+                        <goals>
+                            <goal>build</goal>
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>docker-push</id>
+                        <phase>deploy</phase>
+                        <goals>
+                            <goal>push</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <autoPull>always</autoPull>
+                    <images>
+                        <image>
+                            <name>${dockerCafImagePrefix}opensuse-tomcat${dockerProjectVersion}</name>
+                            <build>
+                                <dockerFileDir>.</dockerFileDir>
+                                 <assembly>
+                                    <basedir>/</basedir>
+                                    <inline>
+                                        <dependencySets>
+                                            <dependencySet>
+                                                <includes>
+                                                    <include>ch.qos.logback:logback-access</include>
+                                                    <include>ch.qos.logback:logback-core</include>
+                                                    <include>com.github.cafapi.logging:caf-logging-logback-access</include>
+                                                    <include>com.github.cafapi.opensuse.tomcat:caf-logging-tomcat-juli</include>
+                                                </includes>
```

Comment:
If you don't want transitive dependencies then just set the flag rather than re-listing out all of the dependencies.